packages/ui/src/components/Tabs/tabs.tsx

```diff
@@ -16,6 +17,10 @@ export type TabItem = {
     children?: React.ReactNode
 }
 
+type CondoTabsTypeProp = {
+    type?: 'rounded'
```

Comment:
Maybe convert this prop to enum? It would be easier in the future to modify it

Reply:
The point is that the default types are not enum and in the future if we want to add them all we have to do is the following

```
import { TabsType } from 'antd/lib/tabs'
...
type CondoTabsTypeProp = {
    type?: 'rounded' | TabsType
}
```

I dont really understand why it would be better to convert CondoTabsTypeProp to enum, can you explain a bit more please